buildSrc/private/src/main/kotlin/androidx/build/AndroidXComposeImplPlugin.kt

```diff
@@ -473,10 +473,6 @@ private fun Project.configureLintForMultiplatformLibrary(
             it.variantInputs.addSourceSets()
         }
 
-        project.tasks.withType<AndroidLintTask>().configureEach {
```

Comment:
Since we don't build/run/publish Android, I think this removal is fine.
Just curious: Is it conflicting with new gradle version?

Reply:
Yes, it caused sync fail. I've checked that it was removed in AOSP too. I did only local removal instead of picking this part with AOSP because it caused a lot more related changes. I'll keep it until full buildSrc sync